scenarios2/10_Lilith.cfg

```diff
@@ -2052,6 +2052,31 @@ As the time comes for two suns to shine brightly in the sky."
             message= _ "Not until I allow it! This should prevent her from dying. We will worry about her later."
         [/message]
     [/event]
+    [event]
+        name=last breath
+        [filter]
+            id=Corruptia
+        [/filter]
+        [message]
+            speaker=unit
+            message= _ "No! Save me! Curse you, Lilith!"
+        [/message]
+        [message]
+            speaker=Efraim
+            message= _ "Among the leaders of Arach she was the only one who turned away from Lilith. She might be helpful to bring local people back to normal life."
+        [/message]
+        [message]
+            speaker=Lethalia
+            message= _ "Anghra-kha'sambghra, sei'ko osstap nalatho, ghelt dob'the ekthanathul."
+        [/message]
+        {MODIFY_UNIT id=Corruptia status.petrified yes}
+        {MODIFY_UNIT id=Corruptia hitpoints 40}
+        {MOVE_UNIT id=Corruptia 42 5}
+        [message]
+            speaker=Lethalia
+            message= _ "I agree. If she dies here, we'll have to restore the order ourselves. Without any help from old leaders it will take a lot of time and effort. This should prevent her from dying. We will worry about her later."
```

Comment:
_It could take a lot of time and effort without the help of the old leadership._